supplier_api_tester/supplier_api_tester/v2/tests/__init__.py

```diff
@@ -41,4 +41,5 @@
 
 PRODUCT_CATALOG = (
     product_catalog.test_get_products,
+    product_catalog.test_products_required_additional_fields,
```

Comment:
Right now it's a separate test so we need to call the catalog endpoint twice, but if we would change our `Product` model
```
    required_order_data: Optional[List[str]] = None
    required_visitor_data: Optional[List[str]] = None
   ```
and use enums then loading that model from JSON should already raise a proper exception that we just need to catch.